ui/src/components/connections/Connections.tsx

```diff
@@ -0,0 +1,179 @@
+import { Avatar, Card, Divider, Row, Space, Table, Tag, Tooltip, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
+import { useCallback, useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { APIError } from "src/adapters/api";
+import { Connection, Credential, getConnections } from "src/adapters/api/connections";
+import { ReactComponent as IconUsers } from "src/assets/icons/users-01.svg";
+import { ConnectionsRowDropdown } from "src/components/connections/ConnectionsRowDropdown";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { NoResults } from "src/components/shared/NoResults";
+import { SiderLayoutContent } from "src/components/shared/SiderLayoutContent";
+import { TableCard } from "src/components/shared/TableCard";
+import { useEnvContext } from "src/contexts/env";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { CONNECTIONS, QUERY_SEARCH_PARAM } from "src/utils/constants";
+import { AsyncTask, isAsyncTaskDataAvailable, isAsyncTaskStarting } from "src/utils/types";
+
+export function Connections() {
+  const env = useEnvContext();
+  const [connections, setConnections] = useState<AsyncTask<Connection[], APIError>>({
+    status: "pending",
+  });
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const queryParam = searchParams.get(QUERY_SEARCH_PARAM);
+
+  const tableContents: ColumnsType<Connection> = [
+    {
+      dataIndex: "userID",
+      ellipsis: { showTitle: false },
+      key: "type",
+      render: (userID: Connection["userID"]) => (
+        <Tooltip placement="topLeft" title={userID}>
+          <Typography.Text strong>{userID.replace(/did:\w*:\w*:\w+:/, "")}</Typography.Text>
```

Comment:
Like this, the tooltip shows the prefix and the column does not, which is strange. Also, I'd consider doing this in the persistence layer and storing the result on a different prop, so the UI can widely and consistently use this stripped version of the did.

Reply:
Probably a function when is used in more places

Reply:
> connection details are going to have different regex and rules

mmm... but then this code will fail or I am missing something?

The user did's I see now have this shape: 

`did:polygonid:polygon:mumbai:2qGGp...Be7kG`

If we can harcode `did:polygonid` we could join the value of the env var `ISSUER_ETHEREUM_RESOLVER_PREFIX` (`polygon:mumbai`) and extract this from the did's. Does it make sense?

And if `did:polygonid` is also going to change, I think we need an env var for it.

This regex does not feel like the way to go.

Reply:
I agree and I think we need more info from API & protocol to come up with a general solution. For now, we should probably just create an issue to track.

Reply:
`did` is going to be there always, I don't know about polygonid, the others are in env-issuer as the RESOLVER_PREFIX. 
I'm not seeing problems with the regex

Reply:
Asked [here](https://0xpolygon.slack.com/archives/C0409DDNV7V/p1680249335844219). I wouldn't create a task if it's something we can learn now. I would want to avoid introducing this regex and have the helper function that given a full DID returns the part that we want to present to the user.

Reply:
As Jordi commented in the thread the DIDs seem to have a different number of segments (3 according to the [ official docs](https://www.w3.org/TR/did-core/#did-syntax)), but since we're only interested in the last one, we can split it with `:` and show the last segment to the user. This is the way to go.

Also, can we please abstract this to a function and use it both in the table and the tooltip?